students/merrix/session02/fizzbuzz.py

```diff
@@ -0,0 +1,9 @@
+for num in range(1, 101):
+    if num % 15 is 0:
```

Comment:
Best not to use `is` when comparing numbers (or almost anything other than `x is None`).